modules/sql/src/main/scala/package.scala

```diff
@@ -27,6 +27,10 @@ package object sql {
   implicit val ZoneIdMeta: Meta[ZoneId] =
     Meta[String].xmap(ZoneId.of, _.toString)
 
-  implicit val MagnitudSystem: Meta[MagnitudeSystem] =
+  implicit val MagnitudSystemMeta: Meta[MagnitudeSystem] =
     Meta[String].xmap(MagnitudeSystem, _.id)
+
+  implicit val MagnitudeValueMeta: Meta[MagnitudeValue] =
```

Comment:
The `Meta` doesn't seem to be used. Why do we need to define it?

Reply:
Yeah the only ones we need are `Duration` and `ZoneId`. You can delete the others.